test/properties_test.cpp

```diff
@@ -44,23 +44,37 @@ TEST(Properties, Precision) {
   EXPECT_FLOAT_EQ(cube.GetEpsilon(), 100 * kPrecision);
 }
 
-TEST(Properties, Precision2) {
+TEST(Properties, Epsilon2) {
   Manifold cube = Manifold::Cube();
   cube = cube.Translate({-0.5, 0, 0}).Scale({2, 1, 1});
   EXPECT_FLOAT_EQ(cube.GetEpsilon(), 2 * kPrecision);
 }
 
-TEST(Properties, Precision3) {
+TEST(Properties, Tolerance) {
   Manifold cylinder = Manifold::Cylinder(1, 1, 1, 1000);
   const auto prop = cylinder.GetProperties();
 
+  Manifold cylinder2 = cylinder.SetTolerance(0.01);
   MeshGL mesh = cylinder.GetMeshGL();
-  mesh.faceID.clear();
-  Manifold cylinder2 = Manifold(mesh).SetEpsilon(0.001);
+  mesh.tolerance = 0.01;
+  Manifold cylinder3(mesh);
+
+  EXPECT_EQ(cylinder.NumTri(), 3996);
+  EXPECT_EQ(cylinder2.NumTri(), 3996);
+  EXPECT_EQ(cylinder3.NumTri(), 3996);
```

Comment:
This is kind of a stupid test right now - the high res cylinder bends too much, so `CreateFaces` gives up and doesn't allow any simplification at all. This was true before too - our `SimplifyTopology` is not a real decimator; it's considerably simpler. However, we should probably come back someday and turn it into a proper decimator.

Meanwhile, I'll update this test to something more useful before we merge this PR.